e2e/tests/Login/SuccessfullyLogIn.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from '@playwright/test';
+
+import {
+  getAccessToken,
+  resetDB,
+} from '../../../services/API-service/test/helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_MALAWI_EMAIL,
```

Comment:
1. I don't think we should load this via env. Instead can we load it from 'users.json' somehow? Or in the same way the utility.helper loads it for API-service? Let's discuss.
2. There's no particular reason to use malawi-specific email. We have a multi-country admin-user 'dunant@redcross.nl' which should be used by default I think. Only to deviate from in specific test cases.

Reply:
1. We can surely use the JSON to load those users if it is more convenient.
2. The reason Malawi was used is that it was so far the only setup that shows correct data in my local environment. For the login test of course it does not matter.

Note:

Tests that will run in the future should probably run per project so different login users should be used. I did not dive so much into that yet but it is food for thought.

Reply:
Use dunant user, and get it from users.json if that's possible, otherwise recreate in e2e folder?

Reply:
Done